azure-toolkit-libs/azure-toolkit-appservice-lib/src/main/java/com/microsoft/azure/toolkit/lib/appservice/config/FunctionAppConfig.java

```diff
@@ -16,5 +16,7 @@ public class FunctionAppConfig extends AppServiceConfig {
     private String appInsightsInstance;
     private String appInsightsKey;
     private boolean disableAppInsights;
+    private boolean nativeExecutable;
```

Comment:
it seems that 'nativeExecutable' and 'customHandlerArgs' are parameters used for package/run right? If so, we could remove them from 'FunctionAppConfig' as 'FunctionAppConfig' is used to create/update FunctionApp in Azure, which will not affect packaging or local run